src/main/java/org/polyjdbc/core/query/InsertQuery.java

```diff
@@ -73,23 +71,11 @@ public InsertQuery into(String tableName) {
      * Insert next sequence value into column of given name. Only one sequenced
      * column per table is supported so far.
      */
-    public InsertQuery sequence(String sequenceField, String sequenceName) {
```

Comment:
no, this API is not changed, the sequence() method is still in the same place :)